server/server.js

```diff
@@ -1,15 +1,98 @@
 const express = require("express");
 const app = express();
 const port = process.env.PORT || 5000;
+const exampledata = require('../exampleresponse.json')
+const {Pool} = require('pg');
+const cors = require ('cors')
+require('dotenv').config({path:'./.env'})
 
+const db = new Pool({
+  user: process.env.DB_USERNAME,
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATABASE,
+  password: process.env.DB_PASSWORD,
+  port: process.env.DB_PORT,
+});
+
+app.use(express.json()); 
 app.listen(port, () => console.log(`Listening on port ${port}`));
 
 // Store and retrieve your videos from here
 // If you want, you can copy "exampleresponse.json" into here to have some data to work with
-let videos = [];
+let videos = exampledata;
 
 // GET "/"
 app.get("/", (req, res) => {
-  // Delete this line after you've confirmed your server is running
-  res.send({ express: "Your Backend Service is Running" });
+  const order = req.query.order || "desc";
+
+  // Sort videos based on the 'order' parameter
```

Comment:
It looks like you're sorting both on the server side and the client side - do you need to be doing both? What advantages/disadvantages does each place to do it have?